app/models/forms/claim.rb

```diff
@@ -4,10 +4,7 @@ class Claim < Base
     attribute :identifier, String
 
     validates :number, inclusion: { in: [true, false] }
-
-    with_options if: :number do
-      validates :identifier, presence: true
-    end
+    validates :identifier, presence: true, length: { maximum: 50 }, if: :number?
```

Comment:
case numbers aren't normally more than 10.  25 should be ample... and free up space